### PR TITLE
Fixed fql call & added new messages :birthday:

### PR DESCRIPTION
**Pull request by boscorona**

### CODE REVIEW

File: thanks.py [hunk 1]
```diff
@@ -20,28 +20,30 @@
     'Thank you!! :D',
     'Thanks so much!',
     'Thanks!',
-    'Thank you! I appreciate it!'
+    'Thank you! I appreciate it!',
+    'Thank you! <3'
```
Comment (Rican7) [inline thanks.py:24]: ha, nice

File: thanks.py [hunk 2]
```diff
@@ -20,28 +20,30 @@
...
-
 # Define our ridiculous "birthday" query
 def build_birthday_fql(min_date, max_date, limit = 500):
     birthday_fql = ("SELECT post_id, actor_id, target_id, created_time, message, comments, like_info "
                     "FROM stream "
                     "WHERE source_id = me() "
-                        "AND filter_key = 'others' "
                         "AND created_time > {} "
                         "AND created_time < {} "
-                        "AND actor_id != me() "
```
Comment (Rican7) [inline thanks.py:38]: why are you removing these query filters?

Comment (boscorona) [inline thanks.py:38]: It caused an error in Graph API, when I removed them, the query worked perfectly.

Comment (Rican7) [inline thanks.py:38]: Hmm, weird. The Graph API must have updated with a BC break.
Classssic Facebook. :disappointed:

**Pull request closed without merging**